src/main/java/com/thebois/models/inventory/Inventory.java

```diff
@@ -27,9 +27,8 @@ public void addMultiple(final List<IItem> stack) {
 
     @Override
     public IItem take(final ItemType itemType) {
-
-        if (hasItem(itemType)) {
-            final int firstIndexOfItemType = getFirstIndexOf(itemType);
+        final int firstIndexOfItemType = getFirstIndexOf(itemType);
+        if (firstIndexOfItemType >= 0) {
```

Comment:
style

IMO `!= -1` makes it more clear that it's checking that an index was found. `>= 0` had me thinking for a split second that this was some sort of count.

Reply:
Just think faster 😏